contracts/Removal.sol

```diff
@@ -299,12 +318,7 @@ contract Removal is
       removals: removals,
       projectId: projectId
     });
-    if (holdbackPercentage > 100) {
-      revert InvalidHoldbackPercentage({
-        holdbackPercentage: holdbackPercentage
-      });
-    }
-    _projectIdToHoldbackPercentage[projectId] = holdbackPercentage;
+    _setHoldbackPercentage(projectId, holdbackPercentage);
```

Comment:
Use named params

Reply:
no worries this is a good and worthy nit!